satpy/scene.py

```diff
@@ -1143,6 +1143,45 @@ def save_datasets(self, writer=None, filename=None, datasets=None, compute=True,
                                           **kwargs)
         return writer.save_datasets(dataarrays, compute=compute, **save_kwargs)
 
+    def compute(self, **kwargs):
+        """Call `compute` on all Scene datasets.
+
+        See :meth:`xarray.DataArray.compute` for more details.
+        """
```

Comment:
```suggestion
        See :meth:`xarray.DataArray.compute` for more details.
        Note that this will convert the contents of the DataArray to numpy arrays which
        may not work with all parts of Satpy which may expect dask arrays.
        """
```